setup.py

```diff
@@ -319,6 +363,6 @@ def get_requirements() -> List[str]:
     python_requires=">=3.8",
     install_requires=get_requirements(),
     ext_modules=ext_modules,
-    cmdclass={"build_ext": BuildExtension},
+    cmdclass={} if _is_neuron() else {"build_ext": BuildExtension},
```

Comment:
nit: Can we change to this since CUDA is more common?
```suggestion
    cmdclass={"build_ext": BuildExtension} if not _is_neuron() else {},
```

Reply:
nit: WDYT about this change?